requirements.txt

```diff
@@ -1,3 +1,4 @@
+GitPython==2.1.12
 pydocstyle==3.*
```

Comment:
the last version is 4.0.0. Why should we use 3.*?

Reply:
Cause have error with last version.
https://github.com/sphericalpm/lektorium/pull/37

Reply:
Ok, then i think it will be better to fixed the last version obviously.